tools/snp.sh

```diff
@@ -275,6 +284,13 @@ install_dependencies() {
   # Needed to build 6.11.0-rc3 SNP kernel on the host
   pip install tomli
   
+  #Needed to build Coconut-SVSM
+  if [ "$SVSM" ]; then
+    sudo apt install -y libcunit1 libcunit1-dev build-essential libclang-dev autoconf \
+      autoconf-archive pkg-config automake libssl-dev libc6-dev gcc-multilib binutils make musl musl-tools
+  fi
```

Comment:
@ryansavino same thoughts about this part too. Should we look at integrating this 
```suggestion
  if $SVSM; then
    sudo apt install -y libcunit1 libcunit1-dev build-essential libclang-dev autoconf \
      autoconf-archive pkg-config automake libssl-dev libc6-dev gcc-multilib binutils make musl musl-tools
  fi
```